roles/edpm_neutron_metadata/molecule/default/converge.yml

```diff
@@ -22,6 +22,7 @@
     - ansible.builtin.include_role:
         name: "osp.edpm.edpm_neutron_metadata"
       vars:
+        ansible_user: root
```

Comment:
what's triggering this change now? which user was used before?

Reply:
I think that previously we didn't use this variable in this role and that's why it wasn't needed.